app/views/recipes/_recipe.html.erb

```diff
@@ -0,0 +1,21 @@
+<li class='card'>
+    <div class='recipe-photo'>
+        <img src='<%= recipe.photo %>' alt='<%= recipe.name %>' class='recipe-photo' />
+        </div>
+        <div class='recipe-info'>
+        <div class='recipe-n'>
+            <%= link_to "#{recipe.name}", recipe_path(recipe), class: 'recipe-name' %>
+            <%= link_to "Remove", recipe_path(recipe), data: {
+                    turbo_method: :delete,
+                    turbo_confirm: 'Are you sure you want to delete this recipe?'
+                }, class: 'remove' if can? :destroy, recipe %>
+        </div>
+        <div>
```

Comment:
- [ ] When trying to delete a recipe that have foods I got **InvalidForeignKey** error (see image).
![Screenshot from 2023-12-23 18-07-49](https://github.com/patriciachrysy/Recipe-App/assets/91142432/3d13f114-b2f5-4bff-9023-6c8b90273c62)
This error can be fixed by adding `dependent: :destroy` to the the recipe and recipe_foods in the recipe model
```rb
    has_many :recipe_foods, dependent: :destroy
```

Reply:
Fixed. Successfully delete the recipe.
![image](https://github.com/patriciachrysy/Recipe-App/assets/93598189/30ae1c04-6f5f-4f3a-b3aa-819307b4315e)